### ISSUE TITLE
JOSS review issues

### ISSUE THREAD

**Issue by srmnitc** [wasserman-group/pyCADMium#10]
This is part of the review for JOSS at https://github.com/openjournals/joss-reviews/issues/4459 .

The following issues were found during the review:

### Installation
- Installation instructions are given on the README, but not on documentation page.
- Installation instructions for `libxc`: The `tar` command is given in the wrong order. It should be `tar` first, then `cd` command. Also, the `wget` command download a file called `down.php?file=5.0.0%2Flibxc-5.0.0.tar.gz`.

### Functionality
- Please see https://github.com/wasserman-group/pyCADMium_examples/issues/3
- Functionality is not clear from the documentation page. The available notebooks can be easily added to the existing documentation.

### A Statement of need
- No statement of need in documentation.
### Installation instructions
- No installation instructions in the documentation
### Example usage
- Examples available in a separate repository, but not in documentation
### Functionality documentation
- Class methods are, for example, in `kohnsham.py`, have no docstrings. When docstrings are present, the arguments and return values are not specified
### Automated tests
- Automated tests exist, but it is not explained anywhere on how to run them properly
- Tests fail because there are packages that are not installed with the installation instructions, but are needed. Eg- `pytest`, `pydantic`
- Not super important, but tests raises a warning with the statement `import numpy.matlib as matlab`
### Community guidelines
- Apart from an email id to contact in case of errors, no other infomation is available.
### Quality of writing
- There are some typos, for example in lines 45, 53, 54, 67. A spellcheck of the whole paper will fix these issues
- lines 10-13 needs a citation for it to be helpful for the general audience
- lines 26-27 "This error can be......eliminated in practice" also needs a citation in my opinion
- line 52, a citation could be helpful
- line 61 acronym P-DFT was not introduced before

**Comment by VHchavez** [comment]
Hello Sarath, 

Thanks for your much helpful comments. I am currently working on your edits on the fly. Otherwise, I find it harder to do it all in one go. I am editing from top to bottom. 

- I have added an analogous installation page to the README in the documentation. 
- I have updated the pylibxc instructions using the much more efficient `pylibxc2` mentioned in the review. 
- I tried the `wget` command and I do get a "clean" download file. Nevertheless, I added a comment so that users are careful in that regard. 

For the functionality, is it enough to add a page that explains the examples as well as links to the example repository, or are you suggesting each of them be added within the documentation. In that case, would adding a block of code suffice? I know that there are smarter ways of embedding `.ipynb` but I am unsure how to do it.

**Comment by srmnitc** [comment]
@VHchavez [nbsphinx](https://nbsphinx.readthedocs.io/en/0.8.9/) can help you embed the notebooks directly. But I think a description of the examples with corresponding links would also be helpful!

**Comment by VHchavez** [comment]
Hello Sarath, 

I've followed your advice on the documentation and added each of the examples as well. I had problems adding `.md` files for some weird reason. I spied on several repositories and I noticed that people add them like normal `.rts`. But I could never figure that out. 

Added the missing information, like the installation to the documentation. 

The KohnSham class now has docstrings to accompany each of its methods. 

Pytest details have been added to the repo's README. And all of the dependencies have been added to the `setup.py`

I encouraged folks to submit issues and pull requests. I am not sure what else can be added. 

Thanks for reviewing the document with the huge amount of spelling errors 🙈. They have been addressed as well as the sentences with no citation.
